src/custom/components/ProgressBar/styled.tsx

```diff
@@ -0,0 +1,75 @@
+import styled from 'styled-components/macro'
+import * as CSS from 'csstype'
+import { FlexWrap as FlexWrapMod } from 'pages/Profile/styled'
+import { transparentize } from 'polished'
+
+export const FlexWrap = styled(FlexWrapMod)`
+  max-width: 100%;
+  align-items: flex-end;
+`
+
+export const ProgressBarWrap = styled(FlexWrapMod)`
+  max-width: 500px; //optional
+  padding-top: 40px;
+  position: relative;
+`
+
+export const ProgressContainer = styled.div`
+  background-color: ${({ theme }) => transparentize(0.61, theme.text1)};
+  height: 16px;
```

Comment:
Hey @maria-vslvn, I love it! Works fine! 
Maybe we can add 24px height in order to have more space inside the bar and increase the font size of the variable %. (to `font-size: 16px;` and `font-weight: 900;`)

Another thing, we can decrease a bit the font-size of the values at the top (to 12px).

It'll look like this:
![image](https://user-images.githubusercontent.com/622217/148568142-9cab5a8c-6577-4af0-8b98-2840e368d1a8.png)